src/scripts.js

```diff
@@ -324,6 +345,32 @@ const toggleFavorites = (recipe, id) => {
   displayAllRecipes(currentRecipes);
 };
 
+const changeStock = (recipe, subtractStock = -1) => {
+  if (checkUserStock(recipe)) {
+    recipe.singleRecipe.ingredients.forEach((recipeIngredient, index) => {
+      this.pantry.forEach((pantryIngredient) => {
```

Comment:
Will need to refer to User here and a few other places in this function instead of "this" - Jordan